flexmeasures/api/v3_0/users.py

```diff
@@ -91,26 +105,80 @@ def index(self, account: Account, include_inactive: bool = False):
         :status 403: INVALID_SENDER
         :status 422: UNPROCESSABLE_ENTITY
         """
+        if (page is None and per_page is not None) or (
```

Comment:
Actually, the best place to put this validation would be in a Marshmallow schema, in api/common/schemas/pagination.py. 
That is a bit of work, but it is nice to get that logic out of the endpoint.

A problem I see is that they are two parameters you want to check not only for values but whether they are both set or none is set. Let's not do that for now. The asset API endpoint simply checks if `page` is set and decides on that information alone whether to use pagination. That is good enough for me.

The right status code for wrong parameter usage is 422